lib/atlassian-jwt-authentication/filters.rb

```diff
@@ -55,11 +55,14 @@ def on_add_on_installed
           if user[:username].present? && user[:display_name].present? &&
               user[:uuid].present? && user[:type].present? && user[:type] == 'user'
 
-            jwt_user = current_jwt_auth.jwt_users.where(user_key: user[:uuid]).first
-            JwtUser.create(jwt_token_id: current_jwt_auth.id,
-                           user_key: user[:uuid],
-                           name: user[:username],
-                           display_name: user[:display_name]) unless jwt_user
+            jwt_user = current_jwt_auth.jwt_users.find_or_initialize_by(user_key: user[:uuid]) do |jwt_user|
+              jwt_user.name = user[:username]
+              jwt_user.display_name = user[:display_name]
+            end
+
+            jwt_user.update!(
+              name: user[:username],
+              display_name: user[:display_name])
```

Comment:
Why update the display_name here, but also set it above? I personally find the sequence of find_or_initialize_by qith a block and update! a bit confusing, I think current_jwt_auth.jwt_users.where plus create was clearer.

Reply:
Yeah, I guess it could be simplified. Done.